motion/joystick_interface/README.md

```diff
@@ -0,0 +1,40 @@
+## Joystick interface
+
+#### Subscribes to:
+* /joystick/joy
+
+#### Publishes to:
+* /joystick/input
+
+This package contains a mapping between the joystick raw data and what we consider to be "inputs". "Inputs are defined as 
+movement of the joysticks in all 6 DOFs.
+
+## Setup
+The joystick may require some configuring before use. Take special note of the **Configuration** step.
```

Comment:
We should add a note about this in the [wiki under software installation ](https://github.com/vortexntnu/Vortex-AUV/wiki/Software-installation)

Reply:
Agreed 👍